scmtfvc.properties

```diff
@@ -2,12 +2,18 @@ category=Integration
 description=Provides SCM TFVC integration
 devVersion=
 privateVersions=
-publicVersions=2.2
+publicVersions=2.3,2.2
 archivedVersions=1.0,2.0,2.1,2.1.1,2.1.2
 
 defaults.mavenGroupId=org.sonarsource.sonar-plugins.tfvc
 defaults.mavenArtifactId=sonar-scm-tfvc-plugin
 
+2.3.description=embed TFS API
+2.3.sqVersions=[7.3,LATEST]
+2.3.downloadUrl=https://github.com/lg2de/sonar-scm-tfvc/releases/download/2.3.0/sonar-scm-tfvc-plugin-2.3.0-73.jar
+2.3.date=2020-02-06
+2.3.changelogUrl=https://github.com/lg2de/sonar-scm-tfvc/releases/tag/2.3.0
+
 2.2.description=Fix compatibility for SQ 7.3 and above
 2.2.sqVersions=[7.3,LATEST]
```

Comment:
Only one version can be compatible with `LATEST` at a time. Best practice is to set the old version's range end to the current version of SQ.